MakeWin.bat

```diff
@@ -0,0 +1,157 @@
+@echo off
+REM Windows script for the SimPhoNy Framework
+
+REM Current features of this script:
+REM     - install simphony-common package
+REM     - install nCad wrapper package
+REM     - install nfluid package
+REM     - install entire SimPhoNy Framework
+REM     - uninstall entire SimPhoNy Framework
+REM     - install pip package manager
+REM     - install numpy extension
+
+REM All installation packages are directly downloaded and installed from GitHub
+
+REM What is pendant in this script:
+REM     - Lammps integration?
+REM     - JYU-LB integration?
+REM     - Openfoam integration?
+REM     - Mayavi integration?
+REM     - Testing command
+REM     - Virtualenv command
+
+
+REM Current SimPhoNy version to be installed
+set SIMPHONYVERSION=0.2.0
+REM Current nCad version to be installed
+set NCADVERSION=0.2.1
+REM Current nFluid version to be installed
+set NFLUIDVERSION=0.0.2.1
+
+
+REM Check that we only get one argument
+if     '%1'==''  goto help
+if not '%2'==''  goto help
+goto check_target
+
+
+:help
+echo Usage: 'MakeWin ^<target^>' with one of these targets:
+echo   simphony:        install simphony library (version %SIMPHONYVERSION%)
+echo   simphony-ncad:   install ncad plugin      (version %NCADVERSION%)
+echo   nfluid:          install nfluid package   (version %NFLUIDVERSION%)
+echo   all:             install simphony, ncad and nfluid modules
+echo   clean:           remove simphony, ncad and nfluid modules
+echo   pip:             install pip package manager
+echo   numpy:           install unofficial numpy extension
+echo.
+echo Please add C:\Python27\Scripts\ to your PATH environment
```

Comment:
Thanks. 
I am sorry I did not explain myself clearly.  I meant to add "C:\Python27\Scripts" for `pip` to be accessible from the command line.